Sources/Shared/UI/Documents/FileActionsViewController.h

```diff
@@ -19,6 +19,7 @@
 
 #import <Foundation/Foundation.h>
 
+#define PreferredContentSize CGSizeMake(200,280)
```

Comment:
By convention (don't know which one), constants start with a 'k' so please rename in `kPreferredContentSize`